src/System.Security.Cryptography.Encoding/ref/System.Security.Cryptography.Encoding.cs

```diff
@@ -66,4 +66,29 @@ public enum OidGroup
         SignatureAlgorithm = 4,
         Template = 9,
     }
+
+    public sealed class AsnEncodedDataCollection : System.Collections.ICollection, System.Collections.IEnumerable
+    {
+        public AsnEncodedDataCollection() { }
+        public AsnEncodedDataCollection(AsnEncodedData asnEncodedData) { }
+        public int Count { get { return default(int); } }
+        public bool IsSynchronized { get { return default(bool); } }
+        public object SyncRoot { get { return default(object); } }
```

Comment:
We've been making `IsSynchronized` and `SyncRoot` explicitly implemented (see OidCollection)